firstvoices/backend/tests/test_apis/test_sites_api.py

```diff
@@ -186,3 +186,142 @@ def test_detail_404(self):
         response = self.client.get(f"{self.get_detail_endpoint('fake-site')}")
 
         assert response.status_code == 404
+
+
+class TestMySites(BaseApiTest):
```

Comment:
These are great but please put them in their own file. my-sites is its own api :)